### PR TITLE
[1LP][RFR][NOTEST] Added manual test case

### PR DESCRIPTION
**Pull request by ganeshhubale**
Added manual test cases for automate.
Also added comment to automate newly added automate methods.

### CODE REVIEW

File: cfme/tests/automate/test_automate_manual.py [hunk 1]
```diff
@@ -866,3 +866,57 @@ def test_null_coalescing_fields():
...
+
+@pytest.mark.tier(1)
+@pytest.mark.ignore_stream("5.10")
+def test_service_retirement_from_automate_method():
+    """
+    Polarion:
+        assignee: ghubale
+        initialEstimate: 1/8h
+        caseposneg: positive
+        startsin: 5.10
```
Comment (lina-nikiforova) [inline cfme/tests/automate/test_automate_manual.py]: I guess it should be `5.11` according to `ignore_stream` mark

File: cfme/tests/automate/test_automate_manual.py [hunk 2]
```diff
@@ -866,3 +866,57 @@ def test_null_coalescing_fields():
...
+        casecomponent: Automate
+        testSteps:
+            1. Create service catalog item and order
+            2. Create a writeable domain and copy ManageIQ/System/Request to this domain
+            3. Create retire_automation_service instance and set meth5 to retire_automation_service.
+            4. Create retire_automation_service method with sample code given at
+               https://bugzilla.redhat.com/show_bug.cgi?id=1700524#c7
```
Comment (lina-nikiforova) [inline cfme/tests/automate/test_automate_manual.py]: Is it possible to add the code here?

File: cfme/tests/automate/test_method.py [hunk 3]
```diff
@@ -33,6 +33,8 @@ def test_method_crud(klass):
         initialEstimate: 1/16h
         tags: automate
     """
+    # ToDo: Update this test case for other types of automate methods like builtin, expression, uri,
```
Comment (lina-nikiforova) [inline cfme/tests/automate/test_method.py]: Required: Please write #TODO as some effort was made in #8588

Comment (ganeshhubale) [reply cfme/tests/automate/test_method.py]: Thanks :)

**Pull request merged**